src/cascadia/WindowsTerminal/WindowUiaProvider.cpp

```diff
@@ -6,22 +6,18 @@
 
 #include "../host/renderData.hpp"
 
-WindowUiaProvider::WindowUiaProvider(Microsoft::Console::Types::IUiaWindow* baseWindow) :
-    WindowUiaProviderBase(baseWindow)
+HRESULT WindowUiaProvider::RuntimeClassInitialize(Microsoft::Console::Types::IUiaWindow* baseWindow)
 {
+    return __super::RuntimeClassInitialize(baseWindow);
 }
 
-WindowUiaProvider::~WindowUiaProvider()
-{
-}
-
-WindowUiaProvider* WindowUiaProvider::Create(Microsoft::Console::Types::IUiaWindow* baseWindow)
+WindowUiaProvider* WindowUiaProvider::Create(Microsoft::Console::Types::IUiaWindow* /*baseWindow*/)
 {
     WindowUiaProvider* pWindowProvider = nullptr;
     //Microsoft::Terminal::TermControlUiaProvider* pScreenInfoProvider = nullptr;
     try
     {
-        pWindowProvider = Microsoft::WRL::Make<WindowUiaProvider>(baseWindow).Detach();
+        //pWindowProvider = new WindowUiaProvider(baseWindow);

```

Comment:
um dead code? or TODO?

Reply:
So, this is all just a part of me deciding if we even need a WindowUiaProvider in Terminal. It's looking like we won't need it but I need to actually remove it at some point. Should I create a separate issue for this and link it?

Reply:
Yea defs

Reply:
#3564